### ISSUE TITLE
Heading tags not affected by font-size

### ISSUE THREAD

**Issue by apgrover** [edrlab/thorium-reader#1508]
## Summary
Setting font-size on h1-h6 tags does not render a visual change. Default h1-h6 font sizes appear instead.
## Scope
Observed in Thorium for PC/Mac v1.6.0
## Expected
Setting font-size: 2.5em on h1-h6 will set each element to the same font size.
<img src="https://user-images.githubusercontent.com/14042519/122102224-36777200-cde3-11eb-8748-aac56ce3cc7f.png" width=70% />

## Observed
font-size: 2.5em is ignored on h1-h6 and default font size appears instead.
<img src="https://user-images.githubusercontent.com/14042519/122102242-3bd4bc80-cde3-11eb-851e-047fba11ab13.png" width=70% />

### HTML
```
			<h1 class="gx">A</h1>
			<h2 class="gx">A</h2>
			<h3 class="gx">A</h3>
			<h4 class="gx">A</h4>
			<h5 class="gx">A</h5>
			<h6 class="gx">A</h6>
```

### CSS
```
.gx {
    font-size: 2.5em;
}
```

Sample file available upon request.

**Comment by danielweck** [comment]
Hello, thank you for reporting this issue.
However, the observed behaviour seems to match the implemented design, as per ReadiumCSS normalization rules. In other words, this is a feature, not a bug.
That being said, could you please explain the use-case where you would like the publisher / authored styles to be preserved without any normalization at all?
For reference, here is the related ReadiumCSS code:
https://github.com/readium/readium-css/blob/583011453612e6f695056ab6c086a2c4f4cac9c0/css/src/ReadiumCSS-default.css#L96-L135

**Comment by danielweck** [comment]
@JayPanoz I am pinging you here, but I totally understand if you don't have the bandwidth to chime in right now :)

**Comment by JayPanoz** [comment]
@danielweck Yep, that’s the normalisation due to some EPUB authors using absolute values for font-size. 

As usual the advice is to use a “native feature” to handle font-size setting if available/possible. Cf. https://github.com/readium/readium-css/issues/34

**Comment by danielweck** [comment]
Issue moved to ReadiumCSS: https://github.com/readium/readium-css/issues/112
